code/game/objects/items/devices/sound_synth.dm

```diff
@@ -68,6 +83,18 @@
 		spam_flag = world.timeofday
 		//to_chat(M, selected_sound) //this doesn't actually go to their chat very much at all.
 
+/obj/item/device/soundsynth/emag_act(mob/user)
+	spark(src,5,FALSE)
+	if(!emagged)
+		emagged = 1
+		sound_list = sound_list_emagged + sound_list
+		selected_sound = "sound/effects/superfart.ogg"
+
+/obj/item/device/soundsynth/examine(mob/user)
+	..()
+	if(emagged)
```

Comment:
Make this a "holding item" check so that only those who possess it can see it

Reply:
Would `If(user.is_holding_item(src) && emagged)` work?

Reply:
Yes, but put emagged check first for micro-optimization

Reply:
Done.